specification/components/r4/schemas/responses/UnsupportedMediaType.yaml

```diff
@@ -1,5 +1,20 @@
-description: Unsupported Media Type
+description: |
+  Where status code 415 (Unsupported Media Type) is returned then an OperationOutcome will be included in the body, as detailed below. 
+  Check diagnostics property for specific information regarding the error.
+  
+  | issue.details.coding.code  | issue.code    | Coding System                                                      | Description                              |
+  | -------------------------- | ------------- | ------------------------------------------------------------------ | ---------------------------------------- |
+  | REC_UNSUPPORTED_MEDIA_TYPE | not-supported | The server refuses to accept the request because the payload format is in an unsupported format.              |
```

Comment:
Coding system remaining: it will be a BaRS code ([we've asked IOPS/BaRS to create one for us](https://nhsd-confluence.digital.nhs.uk/display/ES/IOPS+Catch-up)).
`[BaRS Error Code](https://fhir.nhs.uk/CodeSystem/http-error-codes)`